File: English/Small Factorials/sm.py




def main():
    testes = raw_input()
    x = 0
    while(x < int(testes)):
        numero_fatorial = raw_input()
        print fatorial(int(numero_fatorial))
        x = x+1

def fatorial(x):
    if (x > 0):
        return x* (fatorial(x-1))
    else:
        return 1


main()

File: English/Small Factorials/sm.php
<?php


function main(){
    fscanf(STDIN, "%d\n", $testes);
    $x = 0;
    while($x < $testes){
        fscanf(STDIN, "%d\n", $numero_fatorial);
        echo fatorial($numero_fatorial) . "\n";
        $x++;
    }
}


function fatorial($x){
    if ($x > 0)
        return $x * (fatorial($x-1));
    else
        return 1;
}

main();
